packages/components/pie-cookie-banner/src/cookie-banner.scss

```
@@ -1 +1,106 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+* {
+    margin: 0;
+}
+
+$breakpoint-wide: 600px;
+
+.c-cookieBanner {
+    --cb-font-family: var(--dt-font-interactive-m-family);
+    --cb-font-size: #{p.font-size('--dt-font-body-l-size')};
+    --cb-line-height: calc(var(--dt-font-body-l-line-height) * 1px);
+    --cb-font-weight: var(--dt-font-body-l-weight);
+    --cb-padding-inline: var(--dt-spacing-d);
+    --cb-padding-block: var(--dt-spacing-d);
+    --cb-offset: 0;
+
+    background-color: var(--dt-color-background-dark);
+    color: var(--dt-color-content-inverse);
+    font-family: var(--cb-font-family);
+    font-size: var(--cb-font-size);
+    line-height: var(--cb-line-height);
+    font-weight: var(--cb-font-weight);
+    padding-block-start: var(--cb-padding-block);
+    padding-block-end: var(--cb-padding-block);
+    max-height: 432px;
+    position: fixed;
+    bottom: var(--cb-offset);
+    left: var(--cb-offset);
+    right: var(--cb-offset);
+    border-top-left-radius: var(--dt-radius-rounded-c);
+    border-top-right-radius: var(--dt-radius-rounded-c);
+
+
+    @media (min-width: $breakpoint-wide) and (orientation: landscape) {
+        --cb-padding-inline: var(--dt-spacing-f);
+        --cb-offset: var(--dt-spacing-d);
+
+        max-height: 90%;
+        border-bottom-left-radius: var(--dt-radius-rounded-c);
+        border-bottom-right-radius: var(--dt-radius-rounded-c);
+    }
+}
+
+.c-cookieBanner-title,
+.c-cookieBanner-body,
+.c-cookieBanner-actions {
+    padding-inline-start: var(--cb-padding-inline);
+    padding-inline-end: var(--cb-padding-inline);
+}
+
+.c-cookieBanner-title {
+    --cb-title-font-size: var(--dt-font-heading-s-size--narrow);
+    --cb-title-line-height: var(--dt-font-heading-s-line-height--narrow);
+
+    font-size: #{p.font-size('--cb-title-font-size')};
```

I think it's a little neater to use the mixin when defining the font-size style (outside of assigning to a variable). Completely accept this could just be a personal preference though. So:

```suggestion
    @include p.font-size('--cb-title-font-size');
```